src/pages/computer-science/data-structures/linked-lists/index.md

```diff
@@ -1,6 +1,4 @@
----
-title: Linked Lists
----
+
 ## Linked Lists
```

Comment:
Hi @mtpjr88 

The reason both the Travis and Netlify builds are failing is because of there being no correct title. I see you removed the original title that was also incorrect.

You can refer to [this](https://github.com/freeCodeCamp/guides/blob/master/CONTRIBUTING.md#title) part of the Style Guide on how the title of an article should be formatted.

Each article needs a title or the build fails.

Reply:
how do i correct it?

Reply:
The very first line of a article should be the title. So if you move the entire title to line 1 and move the heading to line 5 it should be fixed

Reply:
The title should be line 1 to line 3. It starts and ends with `---` and the title in the middle line